src/components/dialogs/field-hook.js

```diff
@@ -292,23 +292,31 @@ export const usePrefillNameField = ({
     selectedFile,
     setValue,
     selectedFileOk,
-    createStudyErr,
+    creationError,
     fileCheckedCase,
     touched,
 }) => {
     useEffect(() => {
         if (setValue) {
+            //here selectedFile is a file the user choosed through a picker
             if (
```

Comment:
If the name was prefilled but not touched by the user, we could change it after we change the case file nope ?

Reply:
It's now implemented